simdev/main.py

```diff
@@ -0,0 +1,3 @@
+if __name__ == "__main__":
```

Comment:
if you rename `main.py` to `__main__.py` - you can call a module like `python -m simdev some_args`